### DISCUSSION TITLE
Supporting other BAAS

### DISCUSSION THREAD

**Discussion by marcellintacite** [receevi/receevi#4] [Q&A]
Hello, I am tacit, a web developer, I am also working on whatsapp busness cloud api integration and luckily I came across receivee. Thank you and congratulations for this great work. But nevertheless, I work with Firebase and I would just like to have your guidance on how I can migrate to Firebase since you use Supabase.

**Comment by junedkhatri31** [comment]
Hi @marcellintacite, Receevi currently only supports Supabase. For firebase to work with Receevi, you would have to change all the code related to authentication and database to Firebase. Let me know if you want help with some task with firebase.
